foundry/foundry.py

```diff
@@ -1,38 +1,26 @@
 
+from multiprocessing.sharedctypes import Value
```

Comment:
Is this import doing anything? If not, delete it. If so, we need to talk why we're touching this lower-level part of Python

Reply:
@Aadit-Ambadkar